taskcluster/.shared.yml

```diff
@@ -142,32 +142,32 @@ system:
       namespace: "project.deepspeech.swig.win.amd64.b5fea54d39832d1d132d7dd921b69c0c2c9d5118"
   tensorflow:
     linux_amd64_cpu:
-      url: "https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.tensorflow.pip.r2.2.7ead55807a2ded84c107720ebca61e6285e2c239.1.cpu/artifacts/public/home.tar.xz"
-      namespace: "project.deepspeech.tensorflow.pip.r2.2.7ead55807a2ded84c107720ebca61e6285e2c239.1.cpu"
+      url: "https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.tensorflow.pip.r2.2-submodule.347767452d19a45a6aeb3694e54adce4d945634a.1.cpu/artifacts/public/home.tar.xz"
```

Comment:
don't forget to change `r2.2-submodule` to `r2.2` here and update the sha1. We might want to reset the `1` to `0` if we change sha1 ?